src/components/UI/FormProjectDuration.vue

```diff
@@ -46,27 +44,24 @@ const sinceOptions = computed(() =>
   props.sinceRequired ? options.value : untilOptions.value
 )
 
-// 出力。stringをオブジェクトに変換して出力
-const handleInput = (value: string | undefined, dateType: DateType) => {
-  const [year, semester] = value?.split(' ') ?? [undefined, undefined]
-  const newValue = {
-    year: Number(year),
-    semester: Number(semester) as Semester
+// ラベルとして表示する文字列
+const objectToString = (value: YearWithSemester | undefined) => {
+  if (value === undefined) {
+    return undefined
+  }
```

Comment:
undefined受け取ってるのにundefinedが来たらundefined返すのちょっと違和感ある
ここは変に関数に閉じ込め過ぎずに変更前と同じように↓くらいの方が個人的に見やすいです
逆操作の方も同様
```js
:model-value="model.since ? objectToString(model.since) : undefined"
```